yivi-docs/blog/2025-06-02-age-verification/index.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: 2025-age-verification
+title: Age Verification in the EU, Ambitious goals, missed privacy opportunities
+authors: [dibranmulder]
+tags: [yivi, eudi, age-verification]
+---
+
+## Protecting minors, today and tomorrow
+[The European Commission is preparing a temporary age verification app](https://digital-strategy.ec.europa.eu/en/policies/eu-age-verification#:~:text=This%20initiative%20aims%20to%20allow,pornography%2C%20gambling%2C%20alcohol%2C%20and%20others), aimed at protecting minors on platforms where a minimum age is required—think of online pornography, gambling, alcohol sales, and certain social media services. The need is hardly disputed: in a digital world, it's legitimate to shield minors from harmful content. The Digital Services Act (DSA) already obligates large platforms to take protective measures, and the Commission is stepping in where compliance is lacking. A technical solution to verify age online is therefore urgently needed.
+
+In May 2025, the European Commission launched [formal investigations into major online pornography platforms](https://ec.europa.eu/commission/presscorner/detail/en/ip_25_1339) for potentially breaching the DSA’s rules on protecting minors. Specifically, the Commission suspects that these platforms may not have effective age verification systems in place, allowing minors to access explicit content too easily. These investigations mark the first enforcement steps of this kind under the DSA, sending a strong signal that the EU intends to hold Very Large Online Platforms (VLOPs) accountable. In announcing the probes, the Commission also referenced its ongoing work on a temporary age verification solution. This tool, they noted, could help platforms meet their obligations while respecting users’ privacy and avoiding repeated identity checks.
+
+The proposed app—a sort of mini European Digital Identity (EUDI) wallet—will allow users, starting in July 2025, to prove their age without repeatedly uploading ID documents. After installation, users verify their age once (e.g., through a passport scan, bank app, or eID login). From then on, the app can inform websites whether a user is over 18 without revealing personal data, according to the Commission. This temporary solution is meant to bridge the gap until the full rollout of EUDI wallets expected at the end of 2026. The initiative is commendable: making the internet safer for minors while providing adults with a digital tool to prove age quickly and privately. Yet the implementation raises key concerns—especially in the realm of privacy.
```

Comment:
Agreeing that age verification will be "making the internet safer for minors" really needs a discussion on the points the EFF and EDRI bring up around age verification, or agree a little less on that point of view.